source/ShipInfoDisplay.cpp

```diff
@@ -362,7 +362,7 @@ void ShipInfoDisplay::UpdateOutfits(const Ship &ship, const Depreciation &deprec
 
 	map<string, map<string, int>> listing;
 	for(const auto &it : ship.Outfits())
-		listing[it.first->Category()][it.first->Name()] += it.second;
+		listing[it.first->Category()][it.first->TrueName()] += it.second;
```

Comment:
```suggestion
		listing[it.first->Category()][it.first->Name()] += it.second;
```

Oops, that's my bad. The names are displayed, so `Name()` is correct here.